Nef_S2/include/CGAL/Nef_S2/Sphere_segment.h

```diff
@@ -37,14 +37,15 @@ Sphere_segment_rep() { ps_ = pt_ = Point(); c_ = Circle(); }
 
 Sphere_segment_rep(const Point& p1, const Point& p2,
                    bool shorter_arc=true) :
-  ps_(p1), pt_(p2), c_(Plane_3(p1,p2,Point_3(CGAL::ORIGIN)))
+  ps_(p1), pt_(p2),
+  c_(CGAL::ORIGIN,R_().construct_orthogonal_vector_3_object()(p1,p2,Point_3(CGAL::ORIGIN)))
```

Comment:
Is it correct that the three parameters could be in any order?

Reply:
If the orientation ("sign" along the normal line) of the orthogonal vector is important, then no. The direction of the orthogonal vector will flip depending on the parity of the permutation. You can change to the order `ORIGIN, p1, p2` and it will not change the orthogonal vector.

Reply:
By the way, i found `R_().construct_orthogonal_vector_3_object()(p1,p2,Point_3(CGAL::ORIGIN))` to be marginally faster than `R_().construct_cross_product_vector_3_object()(p1-CGAL::ORIGIN, p2-CGAL::ORIGIN)`

Reply:
I want to change, as I only want to add the overload with ` Origin` as first parameter. Thanks for the explanation about the parity rule.